e2eTests/src/test/java/IngestionTest.java

```diff
@@ -190,7 +201,7 @@ private void triggerIngestMessageFromCEGA()
     String message =
         String.format(
             "{\"type\":\"ingest\",\"user\":\"%s\",\"filepath\":\"/p11-dummy@elixir-europe.org/files/%s\"}",
```

Comment:
```suggestion
    String message = """
    {
        "type": "ingest",
        "user": "%s",
        "filepath": "/p11-dummy@elixir-europe.org/files/%s"
    }
    """.formatted(EGA_BOX_USERNAME, encFile.getName());
```
Purely for the sake of readability and maintainability.

Reply:
Addressed in [2a59f2a](https://github.com/ELIXIR-NO/FEGA-Norway/pull/124/commits/2a59f2af44be312d5c2495b2cf17e58fd5b77791)